code/_onclick/hud/alien_larva.dm

```diff
@@ -16,7 +16,7 @@
 	move_intent = using
```

Comment:
This hud is still named larva_hud, you should change it to diona_hud at this rate